permission_handler_android/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  permission_handler_platform_interface: ^3.7.0
+  permission_handler_platform_interface: ^3.9.0
```

Comment:
Are these necessary? If so we should create a separate PR updating the dependency in each platform implementation package (`permission_handler_android`, `permission_handler_apple` and `permission_handler_windows`) and release those first. And only afterwards move this PR through (containing only the changes needed for the `permission_handler` package.